pytest_lambda/impl.py

```diff
@@ -66,3 +96,80 @@ def insulator(*args, **kwargs):
             # fixture has no parameter defined for the current test."
             name = 'fixture__' + '__'.join(fixture_names)  # XXX: will this conflict in certain circumstances?
             return create_identity_lambda(name, *fixture_names)
+
+    def contribute_to_parent(self, parent: Union[type, ModuleType], name: str, **kwargs):
```

Comment:
hmmm, if this is new, it likely means the LambdaFixture initialization code elsewhere needs some love

Reply:
This is a total copy/paste job from perchweb. Maybe I shoudn't have assumed this class would be the same here as it is there?

Reply:
So should I nix this method, then?